samples/deploy/crd/v1beta1/ApisixPluginConfig.yaml

```diff
@@ -0,0 +1,64 @@
+#
```

Comment:
Is this generated by `make codegen`?

Reply:
No, Currently `make codegen` does not generate CRD